CMSC320FinalProj_ParkEllis.ipynb

```diff
@@ -439,7 +439,10 @@
     "event_arr = ['4x4', '5x5', '6x6', '7x7'] # Names for each event\n",
     "\n",
     "# Create array with time data for each event\n",
-    "time_array = pd.DataFrame().assign(four = df_grouped['best_4'], five= df_grouped['best_5'], six= df_grouped['best_6'],seven= df_grouped['best_7'])\n",
+    "time_array = pd.DataFrame().assign(four = df_grouped['best_4'])\n",
+    "time_array = time_array.assign(five = df_grouped['best_5'])\n",
+    "time_array = time_array.assign(six = df_grouped['best_6'])\n",
+    "time_array = time_array.assign(seven = df_grouped['best_7'])\n",
```

Comment:
I was initially thinking something like:
```
time_array = pd.DataFrame().assign(
                four = df_grouped['best_4'], 
                five = df_grouped['best_5'], 
                six = df_grouped['best_6'],
                seven = df_grouped['best_7']
)
```

But since this is only one minor fix, it can be resolved in the next (hopefully final!!) PR